### DISCUSSION TITLE
Repeater 2400 mehrere SSIDs in verschiedenen VLANs

### DISCUSSION THREAD

**Discussion by 1RandomDev** [Freetz-NG/freetz-ng#1080] [Allgemein]
Hey, habe (leider) einen Fritz Repeater 2400 den ich wenns geht noch irgendwie verwenden will. OpenWRT ist für das Modell leider nicht verfügbar sonst wäre die Fritz Firmware schon lange runter.
Frage mich jetzt ob es eine anständige Möglichkeit gibt mehrere SSIDs zu konfigurieren die dann mit verschiedenen VLANs gebridged sind. Die VLAN Interfaces per SSH zu erstellen sollte ja kein Problem sein und hostapd was anscheinend für die WLAN Funktionalität benutzt wird könnte man ja auch umkonfigurieren. Bin mir dabei aber halt nicht sicher wie ich verhindere dass die orinale Fritz Firmware meine Änderungen überschreibt bzw. wie stabil das ganze dann läuft wenn ich und die restliche Firmware gleichzeitig Änderungen am WLAN vornehmen. Hat irgendwer hier soetwas schon mal probiert und kann seine Erfahrungen teilen?

**Comment by 1RandomDev** [comment]
Wäre doch praktisch wenn man in der Freetz Config die gesamte AVM Software entfernen könnte sodass nur noch das Linux Grundsystem übrig bleibt wo ich dann per Startscripts meine eigene Config laden könnte

**Comment by fda77** [comment]
Das nennt sich dann "openwrt". Je nach Gerät dann ohne closed source treiber für und/oder dect/isdn/fon/dsl/leds etc
Der ganzen jahrzehnte alte Code in den von avm selbst gebastelten daemons kann kein vlan, man kann schon froh sein dass ipv6 mittlerweile (wenn aktuell auch nur teilweise, siehe vpn) unterstützt wird

**Comment by 1RandomDev** [comment]
Gibt halt wie gesagt leider noch kein OpenWRT für den 2400, wahrscheinlich auch wegen Treiber Problemen

**Comment by fda77** [comment]
Die Repeater hab recht wenige Bauteile mit closed-source treibern. Am besten fragst du da nach:
 - https://forum.openwrt.org/t/adding-openwrt-support-for-avm-fritz-wlan-mesh-repeater-2400/80630
 - https://github.com/PolynomialDivision/openwrt/commits/add-fritzrepeater-2400
